src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -53,18 +54,19 @@ export class AboutComponent implements OnInit, OnDestroy {
   }
 
   setVariables() {
+    const NOT_AVAILABLE = $localize`Not available`;
     const project = {} as any;
-    project.user = localStorage.getItem('dashboard_username');
-    project.role = 'user';
+    project.user = localStorage.getItem(LocalStorage.DASHBOARD_USRENAME);
+    project.role = USER;
     if (this.userPermission.read) {
       this.userService.get(project.user).subscribe((data: any) => {
         project.role = data.roles;
       });
     }
     const browser = detect();
-    project.browserName = browser && browser.name ? browser.name : 'Not detected';
-    project.browserVersion = browser && browser.version ? browser.version : 'Not detected';
-    project.browserOS = browser && browser.os ? browser.os : 'Not detected';
+    project.browserName = browser && browser.name ? browser.name : NOT_AVAILABLE;
+    project.browserVersion = browser && browser.version ? browser.version : NOT_AVAILABLE;
+    project.browserOS = browser && browser.os ? browser.os : NOT_AVAILABLE;
```

Comment:
This is could be simplified like this:
```suggestion
   if(browser) {
    project.browserName = browser.?name || NOT_AVAILABLE;
    project.browserVersion = browser?.version || NOT_AVAILABLE;
    project.browserOS = browser?.os : NOT_AVAILABLE;
    }
```

Reply:
Then it will not set the defaults for each of these and would need refactor like separately, and the current approach is okay and clean.